visual-js/.changeset/proud-wolves-argue.md

```diff
@@ -0,0 +1,5 @@
+---
+"@saucelabs/visual": patch
```

Comment:
Shouldn't deprecation be a minor update rather than a patch ?

Reply:
According to semver, it should: https://semver.org/#spec-item-7

Reply:
ok, I changed it